scripts/after-pack.mjs

```
@@ -0,0 +1,97 @@
+
+export default async function afterPack(context) {
+  // `context.arch` is an Arch enum ordinal; its name is the directory spelling
+  // the SDK stages under (process.arch).
+  const { Arch } = await import('electron-builder');
+  const archName = Arch[context.arch];
+  const keep = `${context.electronPlatformName}-${archName}`;
+
+  const root = unpackedRoot(context);
+  const scope = path.join(root, 'node_modules', '@runanywhere');
+  if (!fs.existsSync(scope)) {
+    console.log(`  after-pack: no @runanywhere payload under ${root}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)